### ISSUE TITLE
Is it possible to set the temperature for room through Automation?

### ISSUE THREAD

**Issue by cloudbr34k84** [Swamp-Ig/homeassistant_izone#60]
### Checklist

- [X] I have filled out the template to the best of my ability.
- [X] This only contains 1 feature request (if you have multiple feature requests, open one feature request for each feature request).
- [X] This issue is not a duplicate feature request of [previous feature requests](https://github.com/ludeeus/integration_blueprint/issues?q=is%3Aissue+label%3A%22Feature+Request%22+).

### Is your feature request related to a problem? Please describe.

Just wondering if it's possible to set the temperature for zones in automation in Home Assistant?

I know you can via the dashboard, but if you want to automate room temperature control and have the zone turn off it reached a desired temperature. 

### Describe the solution you'd like

Basically what I mentioned, but it's just a question really

### Describe alternatives you've considered

N/a

### Additional context

N/a

**Comment by Swamp-Ig** [comment]
Yes, please check the HA docs. There's a service call (now called action) to do this common to all climate entities.

**Comment by cloudbr34k84** [comment]
I did check the documents? Saying nothing about setting temperature?
